misc/snapshotter/snapshotter.sh

```diff
@@ -154,3 +154,21 @@ function deploy_snapshotter() {
     configure_snapshotter
     ${COMMANDLINE} &
 }
+
+function cleanup_snapshotter() {
+    pid=$(ps -ef | grep containerd-nydus-grpc | grep -v grep | awk '{print $1}')
```

Comment:
I suspect that you can use `$SNAPSHOTTER_BINARY`. Something like:

```
pid=$(pidof -s "$SNAPSHOTTER_BINARY" || true)
```

BTW, I'm a little confused here... is the snapshotter daemon running on the host or within the daemonset container? If the former then it needs to run the command with `nsenter`, right?

Reply:
It would running in the daemonset.